lib/server/index.js

```diff
@@ -1,19 +1,26 @@
-const express = require(`express`)
-const bodyParser = require(`body-parser`)
-const cors = require(`cors`)
-const contact = require(`../api/index.js`)
+const express = require("express");
```

Comment:
Maybe we should consistent to using backtick ` rather than single or double quote ' / "

Reply:
Based on my research, people worldwide are using single or double quote rather than backtick.
So I thought it's the best if we follow that.

Reply:
Well, okey . Lets merge this pull request

Reply:
Single

Reply:
Json file is always Double quote... It's have issue sometimes if using other than double quote